docs/bundle-changes.md

```diff
@@ -9,4 +9,4 @@ A user can also populate this directory and provide the checkpoint name on the c
 
 
 As of this point, containerd has no other additions to the bundle.
-Runtime state is currently stored in a tmp filesystem like `/run`.
+Runtime state is currently stored in a temporary filesystem like `/run`.
```

Comment:
`tmp` was correct here, you can make it `tmpfs` if you prefer to be exact about it